src/pages/Summary/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Option } from 'catling';
+import { HorizontalBarChart } from 'components/HorizontalBarChart';
+import { GET_MATCH_VOTES, MatchVotesQuery } from 'graphql/queries';
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router';
+import { Vote } from 'types/gql';
+
+interface Props extends RouteComponentProps<{ id: string }> {}
+
+export const Summary = (props: Props) => (
+  <MatchVotesQuery
+    query={GET_MATCH_VOTES}
+    variables={{ id: props.match.params.id }}
+  >
+    {({ data, loading }) => {
+      if (loading) {
+        return <span>Loading...</span>;
+      }
+
+      const matchVotesToPlayerTotal = (
+        players: {},
+        vote: Vote,
+      ): {} | { [playerId: string]: { name: string; votes: number } } => {
+        if (!players[vote.player.id]) {
+          players[vote.player.id] = {
+            name: vote.player.name,
+            votes: 1,
+          };
+
+          return players;
+        }
+
+        return {
+          ...players,
+          [vote.player.id]: {
+            name: vote.player.name,
+            votes: players[vote.player.id].votes + 1,
+          },
+        };
+      };
+
+      const votes = Option(data)
+        .flatMap(d => Option(d.match))
+        .flatMap(m => Option(m.votes));
+
+      const totalMotmVotesPerPlayer = votes
+        .map(v => v.filter(vote => vote.type === 'motm'))
+        .map(v => v.reduce(matchVotesToPlayerTotal, {}));
+
+      const totalDotdVotesPerPlayer = votes
+        .map(v => v.filter(vote => vote.type === 'dotd'))
+        .map(v => v.reduce(matchVotesToPlayerTotal, {}));
+
+      return (
+        <>
+          <HorizontalBarChart data={totalMotmVotesPerPlayer} type="motm" />
+          <HorizontalBarChart data={totalDotdVotesPerPlayer} type="dotd" />
```

Comment:
Pretty close here. `totalMotmVotesPerPlayer` and `totalDotdVotesPerPlayer` are still Options so you need to `map` to use the value:

```suggestion
          {totalDotdVotesPerPlayer.map(d => <HorizontalBarChart data={d} type="dotd" />).get()}
```

Reply:
I've defaulted to using the method in `ViewVotes/index` as this, while not throwing any TS errors, was throwing a lot of React children errors. Very helpful either way!